vector/src/main/java/im/vector/activity/VectorAppCompatActivity.kt

```diff
@@ -45,7 +46,7 @@ import org.matrix.androidsdk.core.Log
 /**
  * Parent class for all Activities in Vector application
  */
-abstract class VectorAppCompatActivity : AppCompatActivity() {
+abstract class VectorAppCompatActivity : BaseMvRxActivity() {
```

Comment:
You are not using MvRx 1.3? (last version)

Reply:
Will introduce breaking changes

Reply:
ok